ui/app/models/database/role.js

```diff
@@ -69,9 +70,10 @@ export default class RoleModel extends Model {
   rotation_period;
   @attr({
     label: 'Skip initial rotation',
-    editType: 'boolean',
+    editType: 'toggleButton',
     defaultValue: false,
```

Comment:
is it worth adding a comment here that the `defaultValue` is set again depending on the parent database's value in the form?